viewer/packages/api/src/api-helpers/Image360ApiHelper.ts

```diff
@@ -223,6 +224,21 @@ export class Image360ApiHelper {
     this._needsRedraw = true;
   }
 
+  public getCurrentlyEnteredImageInfo(): Image360WithCollection | undefined {
```

Comment:
Would be nice have better name could be `getActiveImageInfo`.

Reply:
agree with Pramod